lib/block.mli

```diff
@@ -94,6 +94,9 @@ val set_variables: t -> (string * string) list
 val unset_variables: t -> string list
 (** [unset_variable t] is the list of environment variable to unset *)
 
+val required_libraries: t -> (Library.Set.t, string) Result.result
```

Comment:
This function does not need to use `Library.Set`. It should return an `string list` and let the calling code handle this.

Reply:
My reasoning was that actually parsing library names here is more future proof and could later be used to load those libraries rather than simply adding a dependency on the package.

I'm not sure whether you're suggesting to move the parsing to `Library.t` values and the building of the set elsewhere or not to do this at all.

In the former case I'm not sure I understand why it would be better but I can move it if you'd like.